sync/class.jetpack-sync-module-attachments.php

```diff
@@ -23,6 +23,7 @@ function send_attachment_info( $attachment_id ) {
 		 * @param int The attachment ID
 		 * @param object The attachment
 		 */
-		do_action( 'jetpack_sync_save_add_attachment', $attachment_id, $attachment );
+		$filter = current_filter();
+		do_action( 'jetpack_sync_save_add_attachment', $attachment_id, $attachment, $filter );
```

Comment:
I think it would be more semantic and easier to search if we sync two different actions, what do you think?